pkg/operation/botanist/component/clusterautoscaler/cluster_autoscaler.go

```diff
@@ -454,6 +448,11 @@ func (c *clusterAutoscaler) computeShootResourcesData(serviceAccountName string)
 					Resources: []string{"jobs", "cronjobs"},
 					Verbs:     []string{"get", "list", "watch"},
 				},
+				{
+					APIGroups: []string{""},
+					Resources: []string{"namespaces"},
+					Verbs:     []string{"get", "list", "watch"},
+				},
```

Comment:
Nit:

```diff
--- a/pkg/operation/botanist/component/clusterautoscaler/cluster_autoscaler.go
+++ b/pkg/operation/botanist/component/clusterautoscaler/cluster_autoscaler.go
@@ -409,7 +409,7 @@ func (c *clusterAutoscaler) computeShootResourcesData(serviceAccountName string)
                                },
                                {
                                        APIGroups: []string{""},
-                                       Resources: []string{"pods", "services", "replicationcontrollers", "persistentvolumeclaims", "persistentvolumes"},
+                                       Resources: []string{"namespaces", "pods", "services", "replicationcontrollers", "persistentvolumeclaims", "persistentvolumes"},
                                        Verbs:     []string{"watch", "list", "get"},
                                },
                                {
@@ -448,11 +448,6 @@ func (c *clusterAutoscaler) computeShootResourcesData(serviceAccountName string)
                                        Resources: []string{"jobs", "cronjobs"},
                                        Verbs:     []string{"get", "list", "watch"},
                                },
-                               {
-                                       APIGroups: []string{""},
-                                       Resources: []string{"namespaces"},
-                                       Verbs:     []string{"get", "list", "watch"},
-                               },
                        },
                }

```

Ref https://github.com/kubernetes/autoscaler/blob/226784a223a2bc3a5c9edde714bbb6b6cc5094fb/charts/cluster-autoscaler/templates/clusterrole.yaml#L47-L59

Reply:
done, thanks for the thorough look!